molecule/command/init/role.py

```diff
@@ -107,8 +124,15 @@ def execute(self):
     type=click.Choice(config.molecule_verifiers()),
     default='testinfra',
     help='Name of verifier to initialize. (testinfra)')
+@click.option(
+    '--template',
+    '-t',
+    type=click.Path(
+        exists=True, dir_okay=True, readable=True, resolve_path=True),
+    help="Path to a cookiecutter custom template to initialize the role. "
+    "The molecule folder will be added to this template")
```

Comment:
The upstream molecule folder will be added to this template

Might clarify it a bit further

Reply:
Done